conda-recipe/meta.yaml

```diff
@@ -1,56 +1,60 @@
+# NOTE: This script has been automatically generated by github.com/ericdill/conda-skeletor
+
 package:
   name: metadatastore
   version: {{ environ['GIT_DESCRIBE_TAG'] }}.post{{ environ['GIT_DESCRIBE_NUMBER'] }}
 
 source:
   git_url: ../
+
   patches:
     - config.patch
-
 build:
   number: 0
 
-  # Note that this will override the default build string with the Python
-  # and NumPy versions
   string: {{ environ.get('GIT_BUILD_STR', '') }}_py{{ py }}
 
+  # TODO add entry points
 
 requirements:
   build:
     - python
-    - six
-
   run:
     - python
-    - mongoengine ==0.8.7
-    - pymongo ==2.9
-    - six
-    - pyyaml
-    - prettytable
-    - humanize
-    - numpy
-    - pytz
-    - jinja2
     - boltons
```

Comment:
Unless I misunderstand, build dependencies are only those things that need to be installed for setup.py to run. Since the only imports in setup.py are setuptools/distutils and versioneer, I am confident that we do not need six.  However, as I type this I am not sure if we need setuptools/distutils or not